modules/authentication/src/handlers/common.ts

```diff
@@ -129,6 +130,14 @@ export class CommonHandlers implements IAuthenticationStrategy {
     return this.logOut(call);
   }
 
+  async updateUser(call: ParsedRouterRequest): Promise<UnparsedRouterResponse> {
+    const { user } = call.request.context;
+    const fields = call.request.params.fields;
```

Comment:
fetch the data from the body specifically, don't rely on params (we should have deprecated that tbh)